LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2011-2022 Massachusetts Institute of Technology, UChicago Argonne
+LLC, and OpenMC contributors
```

Comment:
Looks like this repo started in 2019, so the copyright line should reflect that (also no need to list MIT since I was already at Argone)
```suggestion
Copyright (c) 2019-2022 UChicago Argonne LLC and contributors
```